canonical-web-and-design/ubuntu.com
# Update contact form

## Done

- Updated contact form on `/kubernetes`, `/kubernetes/what-is-kubernetes`, `/kubernetes/compare`, `/kubernetes/install`, and `/kubernetes/managed`
- copydoc: https://docs.google.com/document/d/1u6IkVfDNNwdz4XgHU6FSCnUFfVtfTMNMmtINIPT31cU/edit#

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-10368.demos.haus/kubernetes
- Visit all relevant pages (see above) and check confirm the titles match the copydoc
- Try to submit without filling out filling all relevant fields, check that it correctly fails
- Do the same but with all fields filled and submit 


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4429

## Screenshots

[If relevant, please include a screenshot.]
